src/extension.ts

```diff
@@ -187,7 +194,8 @@ class ExtensionManager implements vscode.Disposable {
       }
     }
     const telemetryProperties: telemetry.Properties = {
-      isMultiRoot: `${isMultiRoot}`
+      isMultiRoot: `${isMultiRoot}`,
+      isFullyActivated: `await ${this.workspaceHasCMakeProject()}`
```

Comment:
I think the `await` has to be in the ${}, but it might be safer just to add a temp variable for this.